src/Tasks/Common/Resources/Strings.resx

```diff
@@ -350,9 +350,8 @@ The following are names of parameters or literal values and should not be transl
 {1} - Restored version of platform package
 {2} - Current version of platform package</comment>
   </data>
-  <data name="UnableToUsePackageAssetsCache" xml:space="preserve">
-    <value>NETSDK1062: Unable to use package assets cache due to I/O error. This can occur when the same project is built more than once in parallel. Performance may be degraded, but the build result will not be impacted.</value>
-    <comment>{StrBegin="NETSDK1062: "}</comment>
+  <data name="UnableToUsePackageAssetsCache_Info" xml:space="preserve">
```

Comment:
What's the reason for removing the id-prefix from the message?  Is it because it's not an error?

Reply:
Yes, I have found that the code creates too much jarring line noise that looks like an error (also adds target file path and line number).

See #2468, which I fixed.

I made debug builds throw when we put codes on to informational messages. Once it got passed the nullref crash, it hit that next.

Reply:
My only concern is it makes it harder to google for the curious, but thanks for the explanation.